src/page_body/structure/blocks/tokens/values/page_block_token_properties.pr

```diff
@@ -3,7 +3,16 @@
 {[ if token.properties ]}
     <div class="custom-properties">
         {[ if (token.value.referencedToken !== null) ]}
-            <div class="custom-property-item"><span class="label">Alias of:&nbsp;</span><span class="custom-value">{{ token.value.referencedToken.origin.name }}</span></div>
+            <div class="custom-property-item">
+                <span class="label">Reference:&nbsp;</span>
+                <span class="custom-value">
+                    {[ if (token.value.referencedToken.origin !== null) ]}
```

Comment:
It is enough to write !token.value.referenceToken.origin because this way you are only testing for null, but in some cases, you need to test for "undefined" as well